apply_extra.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env sh
```

Comment:
Most if not all extra-data applications inline it into manifest.

Reply:
I prefer to keep it separated. More maintainable for me than having it hidden as a bunch of strings inside a json file.

Reply:
@barthalion Do you still require that I inline this file ?

Reply:
It's not a hill I'm willing to die on.